### PR TITLE
chore: fix vagrant vmware link

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksfix-vmware-downloads-link-hashicorp.vercel.app/vagrant/docs/providers/vmware/installation) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1202794559462712) 🎟️

## 🗒️ What

This link is broken in content and is a one-off compared to other docs links since we did restructure the path.

## 🧪 Testing

- [Visit this page](https://dev-portal-git-ksfix-vmware-downloads-link-hashicorp.vercel.app/vagrant/docs/providers/vmware/installation) and click the vmware utility downloads page link. It should resolve to the correct dev portal path.

### CODE REVIEW

File: src/views/docs-view/utils/product-url-adjusters.ts [hunk 1]
```diff
@@ -111,6 +111,11 @@ export function rewriteDocsUrl(
 		(basePath: string) => inputUrl.startsWith(`/${basePath}`)
 	)
 	if (isCurrentProductDocsUrl) {
+		// The vagrant vmware utility downloads page is a unique case, where we did
+		// adjust the url structure in the transition to devdot
+		if (currentProduct.slug === 'vagrant' && inputUrl.startsWith('/vmware')) {
+			return `/${currentProduct.slug}/downloads/vmware`
+		}
```
Comment (BRKalow) [inline src/views/docs-view/utils/product-url-adjusters.ts:118]: are there any other paths under `/vmware/` we need to consider?

Comment (kendallstrautman) [reply src/views/docs-view/utils/product-url-adjusters.ts:118]: No just this one downloads page. The vmware documentation lives under [/docs/providers/vmware](https://dev-portal-git-staging-hashicorp.vercel.app/vagrant/docs/providers/vmware)

Comment (kendallstrautman) [reply src/views/docs-view/utils/product-url-adjusters.ts:118]: Let me double check though

Comment (kendallstrautman) [reply src/views/docs-view/utils/product-url-adjusters.ts]: I adjusted it to only rewrite for that specific link. As far as I could see there are no other /vmware pages though

**Review by BRKalow — APPROVED** [review]
Works 4 me!

**Pull request merged**